org.mozilla.Thunderbird.json

```diff
@@ -0,0 +1,92 @@
+{
+    "app-id": "org.mozilla.Thunderbird",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "thunderbird",
+    "rename-icon": "thunderbird",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* OpenGL access */
+        "--device=dri",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Access to thunderbird configuration and cache files */
+        "--filesystem=~/.cache/thunderbird", "--filesystem=~/.thunderbird",
+        /* Access to home directory (for attachments) */
+        "--filesystem=home",
+        /* Needed for dconf to work */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "modules": [
+        {
+            "name": "autoconf-2.13",
```

Comment:
I doubt it's needed after build, `"cleanup": [ "*" ]` should remove it.

Reply:
Should I update the version of autoconf used?

Reply:
No, it needs 2.13.